zcncore/get_data.go

```diff
@@ -213,7 +220,7 @@ func GetSharders(active, stakable bool, limit, offset int) ([]byte, error) {
 		return nil, err
 	}
 
-	return client.MakeSCRestAPICall(MinerSmartContractAddress, GET_MINERSC_SHARDERS, Params{
+	return node.MakeSCRestAPICallToSharder(MinerSmartContractAddress, GET_MINERSC_SHARDERS, Params{
```

Comment:
Same here and in multiple places in same file.

Reply:
Same reason

Reply:
https://github.com/0chain/gosdk/pull/1685#discussion_r1855204990